frontend/src/main.ts

```diff
@@ -24,8 +24,11 @@ library.add(
 	faLock, faCrown, faUserAstronaut, faVolumeXmark, faBan, faMagnifyingGlass
 )
 
+const pinia = createPinia()
+pinia.use(({ store }) => {store.router = markRaw(router) })
```

Comment:
Can you explain me this tomorrow ?

Reply:
Yep. It comes from [here](https://stackoverflow.com/a/70905178)
This is apparently a necessary step in order to use the vue-router in a store. I had in mind to implement all necessary functions in the game store, but ended up to write everything in the `JoinAGame.vue`.
We could still move everything to the store, which would be cleaner